CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
+## 1.5.1 2020-01-27
+
+-   **Feature** Add support for Session Token in endpoint definition
```

Comment:
Changelog entries are made with the `newChange` npm script, please see the section in `CONTRIBUTING` for that

Reply:
reverted changelog.md changes and ran newchange script